packages/view/backend/sirius-components-view-diagram-edit/src/main/resources/icons/full/obj16/BrightenTool.svg

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
None of those svg files belong in the support for diagrams in the view DSL since nothing is using them here. They should be with the pin icon.

Reply:
done